pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter.go

```diff
@@ -44,6 +45,8 @@ type MigrationCreateAdmitter struct {
 	VirtClient    kubecli.KubevirtClient
 }
 
+var _ validating_webhooks.Admitter = &MigrationCreateAdmitter{}
```

Comment:
Adding this explicit deceleration makes this file dependent on  "kubevirt.io/kubevirt/pkg/util/webhooks/validating-webhooks".
IMO it is not necessary, please see #12439.

Reply:
done